### ISSUE TITLE
Solution

### ISSUE THREAD

**Issue by Youcef3939** [aboy4321/German-Credit#2]
This is underwhelming, your model might be learning sthg but it's not strong 
You need to check for underfitting and re evaluate your data preprocessing and class balance 
You can also possibly tune hyperparameters such as learning rate and batch size etc
This pushing it to 97% accuracy

**Comment by aboy4321** [comment]
I'll check it out when I have the time, thank you.

**Comment by Youcef3939** [comment]
No need to thank me:))
I got here from the tiktok acc lol I left my IG there if u wanna dm or sthg 
Keep going bro! Am sure you'll make it
